apps/showcase/src/components/showcase/configuration/configuration-pres.config.ts

```diff
@@ -36,6 +36,7 @@ export interface ConfigurationPresConfig extends Configuration {
    * @o3rWidgetParam allDestinationsDifferent true
    * @o3rWidgetParam atLeastOneDestinationAvailable true
    * @o3rWidgetParam destinationPattern "[A-Z][a-zA-Z-' ]+"
+   * @o3rRestrictionKeys "extremely sensitive config"
```

Comment:
Should we name the tag `@o3rRestrictionKey` as if we want several keys we will have several `@o3rRestrictionKey`